### PR TITLE
translation(pt-br): Translate docs concepts section

### PR DESCRIPTION
**Pull request by felipesuri**
Continuing Portuguese Brazil translation of the docs. (#1987)

- [x] Translate all `docs/2.concepts` section.

### CODE REVIEW

**Comment by felipesuri** [comment]
Sorry, this PR has commits that have already been merged, can someone help me how to create a PR without showing these commits?

Or is it ok to do the PR like this?

**Comment by smarroufin** [comment]
Hi @felipesuri 
Thanks for those translations!

It seems you have some missing commits as I've updated many files to unify english translations [be47155](https://github.com/nuxt/nuxtjs.org/commit/be47155d53db16cc3d8f3c37099b0d6186f28ae2) and I see at least one file rolled back ([content/pt-br/docs/2.concepts/5.nuxt-lifecycle.md](https://github.com/nuxt/nuxtjs.org/pull/1995/files#diff-5a7172b8eb7c9e4d8685bcada256645e6b8cdebd9dcde4f36845e503cf639d24)).
As if you merged `main` but it wasn't up-to-date.

Make sure your `main` is up-to-date, then on your PR do a `git rebase main` (, resolve conflicts) and then `git push --force`. This should be good.

Also you have something "prettyfying" your markdown files and since the MDC syntax is not properly handled, it breaks files: [content/pt-br/tutorials/1.creating-blog-with-nuxt-content.md](https://github.com/nuxt/nuxtjs.org/pull/1995/files#diff-bacbfe4a0707e7566dff5b06bf5a7877409f8763c295b60a4dc629dd307416d8)

**Comment by felipesuri** [comment]
Thanks, @smarroufin!

- I've put the missing commits
- I've made `git rebase main`
- I've disabled the [Markdown All in One](url) VSCode extension, this causing bug on [content/pt-br/tutorials/1.creating-blog-with-nuxt-content.md](https://github.com/nuxt/nuxtjs.org/pull/1995/files#diff-bacbfe4a0707e7566dff5b06bf5a7877409f8763c295b60a4dc629dd307416d8)

I think this should be good.

File: content/pt-br/deployments/github-pages.md [hunk 1]
```diff
@@ -19,12 +20,17 @@ Nuxt gives you the possibility to host your web application on any static hostin
...
 ```bash [Yarn]
 yarn generate
 ```
+
 ```bash [NPM]
 npm run generate
 ```
+
+::
+
```
Comment (smarroufin) [inline content/pt-br/deployments/github-pages.md:33]: Several component closings (`::`) have been added in this file, it seems wrong

File: content/pt-br/announcements/5.nuxt3-beta.md [hunk 2]
```diff
@@ -73,7 +74,7 @@ Here some major milestones we've achieved on the way to Nuxt 3:
 - **Sep 5, 2021**: Init [Nuxt Bridge](https://v3.nuxtjs.org/getting-started/bridge) for improving Nuxt 2 experience
 - **Sep 7, 2021**: Support Vite build for production
 - **Oct 11, 2021**: Add [`useState`](https://v3.nuxtjs.org/docs/usage/state) and [`useFetch`](https://v3.nuxtjs.org/docs/usage/data-fetching#usefetch) composables
-::
+  ::
```
Comment (smarroufin) [inline content/pt-br/announcements/5.nuxt3-beta.md:77]: Keep the `::` at the same indentation as the `::component`

File: content/pt-br/teams/index.md [hunk 3]
```diff
@@ -15,19 +15,18 @@ Teams
...
 ::
 
-::team-section
----
-teamName: community
----
+## ::team-section
+
+## teamName: community
+
```
Comment (smarroufin) [inline content/pt-br/teams/index.md:29]: This file also has some broken markdown

**Review by smarroufin — CHANGES_REQUESTED** [review]

**Comment by felipesuri** [comment]
@smarroufin I solved these problems.

File: content/pt-br/announcements/5.nuxt3-beta.md [hunk 4]
```diff
@@ -5,22 +5,22 @@ description: "468 days after the first commit, the Nuxt 3 beta has finally arriv
...
-    avatarUrl: https://pbs.twimg.com/profile_images/1326211963614007302/UJyvtK2f_400x400.jpg
-    link: https://twitter.com/danielcroe
-  - name: Alexandre Chopin
-    avatarUrl: https://pbs.twimg.com/profile_images/1255027239458091009/qMp_q8vy_400x400.jpg
-    link: https://twitter.com/IAmNuxt
-tags:
-  - framework
-  - release
-  - beta
+- name: Sébastien Chopin
```
Comment (ChristopheCVB) [inline content/pt-br/announcements/5.nuxt3-beta.md:8]: Please keep original indentation

File: content/pt-br/announcements/5.nuxt3-beta.md [hunk 5]
```diff
@@ -57,6 +57,7 @@ Check out the community section of the Nuxt 3 website for [getting help](https:/
 Here some major milestones we've achieved on the way to Nuxt 3:
 
 ::list
+
```
Comment (ChristopheCVB) [inline content/pt-br/announcements/5.nuxt3-beta.md:60]: remove empty line

File: content/pt-br/deployments/fume.md [hunk 6]
```diff
@@ -5,9 +5,10 @@ description: How to deploy Nuxt on Fume?
 target: Static & Server
 category: deployment
 logo:
-  light: "/img/companies/square/light/Fume.svg"
-  dark: "/img/companies/square/dark/Fume.svg"
+  light: '/img/companies/square/light/Fume.svg'
+  dark: '/img/companies/square/dark/Fume.svg'
 ---
+
```
Comment (ChristopheCVB) [inline content/pt-br/deployments/fume.md:11]: remove empty line

File: content/pt-br/deployments/fume.md [hunk 7]
```diff
@@ -33,12 +34,15 @@ Get a production-ready URL in 2 minutes with these steps:
 - Run the following command inside your projects root folder
 
 ::code-group
+
```
Comment (ChristopheCVB) [inline content/pt-br/deployments/fume.md:37]: remove empty line

File: content/pt-br/deployments/fume.md [hunk 8]
```diff
@@ -33,12 +34,15 @@ Get a production-ready URL in 2 minutes with these steps:
...
 ```bash [Yarn]
 yarn global add fume-cli
 fume deploy
 ```
+
```
Comment (ChristopheCVB) [inline content/pt-br/deployments/fume.md:42]: remove empty line

File: content/pt-br/deployments/fume.md [hunk 9]
```diff
@@ -33,12 +34,15 @@ Get a production-ready URL in 2 minutes with these steps:
...
 ```bash [NPM]
 npm install -g fume-cli
 fume deploy
 ```
+
```
Comment (ChristopheCVB) [inline content/pt-br/deployments/fume.md:47]: remove empty line

File: content/pt-br/deployments/github-pages.md [hunk 10]
```diff
@@ -5,9 +5,10 @@ description: How to deploy Nuxt app on GitHub Pages?
 target: Static
 category: deployment
 logo:
-  light: "/img/companies/square/light/Github_Pages.svg"
-  dark: "/img/companies/square/dark/Github_Pages.svg"
+  light: '/img/companies/square/light/Github_Pages.svg'
+  dark: '/img/companies/square/dark/Github_Pages.svg'
 ---
+
```
Comment (ChristopheCVB) [inline content/pt-br/deployments/github-pages.md:11]: remove empty line

File: content/pt-br/deployments/github-pages.md [hunk 11]
```diff
@@ -19,12 +20,15 @@ Nuxt gives you the possibility to host your web application on any static hostin
 To deploy on GitHub Pages, you need to generate your static web application:
 
 ::code-group
+
```
Comment (ChristopheCVB) [inline content/pt-br/deployments/github-pages.md:23]: remove empty line

File: content/pt-br/deployments/github-pages.md [hunk 12]
```diff
@@ -19,12 +20,15 @@ Nuxt gives you the possibility to host your web application on any static hostin
...
 ```bash [Yarn]
 yarn generate
 ```
+
```
Comment (ChristopheCVB) [inline content/pt-br/deployments/github-pages.md:27]: remove empty line

File: content/pt-br/deployments/github-pages.md [hunk 13]
```diff
@@ -19,12 +20,15 @@ Nuxt gives you the possibility to host your web application on any static hostin
...
 ```bash [NPM]
 npm run generate
 ```
+
```
Comment (ChristopheCVB) [inline content/pt-br/deployments/github-pages.md:31]: remove empty line

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Please fix indentation and useless empty lines where ever you changed it.

File: content/pt-br/docs/2.concepts/3.server-side-rendering.md [hunk 14]
```diff
@@ -1,23 +1,24 @@
 ---
-title: Server Side Rendering
-description: Server-side rendering (SSR), is the ability of an application to contribute by displaying the web-page on the server instead of rendering it in the browser.
+title: Renderização do Lado do Servidor
+descrição: Renderização do lado do servidor (SSR), é a capacidade de uma aplicação renderizar uma página web no servidor em vez de renderizá-la no navegador.
```
Comment (ChristopheCVB) [inline content/pt-br/docs/2.concepts/3.server-side-rendering.md:3]: Properties are never to be translated -> `description`

File: content/pt-br/docs/2.concepts/5.nuxt-lifecycle.md [hunk 15]
```diff
@@ -1,108 +1,106 @@
 ---
-title: Nuxt Lifecycle
-description: No matter which tool you use, you will always feel more confident when you understand how the tool works under the hood. The same applies to Nuxt.
+title: O Ciclo de Vida do Nuxt
+descrição: Não importa qual ferramenta você use, você sempre se sentirá mais confiante quando entender como a ferramenta funciona por baixo dos panos. O mesmo se aplica ao Nuxt.
```
Comment (ChristopheCVB) [inline content/pt-br/docs/2.concepts/5.nuxt-lifecycle.md:3]: Properties are never to be translated -> description

**Comment by felipesuri** [comment]
@ChristopheCVB It's solved. The bug on `content/pt-br/announcements/5.nuxt3-beta.md` too, and now I can see the announcements on the `pt-br` site translation.

**Review by smarroufin — APPROVED** [review]
LGTM!

File: content/pt-br/docs/2.concepts/2.context-helpers.md [hunk 16]
```diff
@@ -114,52 +116,52 @@ export default {
...
 
-You can do so, by using `this.$nuxt.refresh()`!
+Você pode fazer isso, usando `this.$nuxt.refresh()`
 
 ```html
 <template>
   <div>
     <div>{{ content }}</div>
-    <button @click="refresh">Refresh</button>
+    <button @click="refresh">Atualizado</button>
```
Comment (ChristopheCVB) [inline content/pt-br/docs/2.concepts/2.context-helpers.md:157]: Atualizar

File: content/pt-br/docs/2.concepts/4.static-site-generation.md [hunk 17]
```diff
@@ -1,41 +1,43 @@
...
+### Passo 3: Navegador para Navegador
 
-Navigating between pages with [`<NuxtLink>`](/docs/features/nuxt-components#the-nuxtlink-component) is done on the client side so you don't hit the CDN again and all API calls will be loaded from the already cached static folder even if you hard refresh the browser.
+A Navegação entre as páginas com [`<NuxtLink>`](/docs/features/nuxt-components#the-nuxtlink-component) é feita no lado do cliente então a CDN não será chamada novamente e todas as chamadas à APIs serão carregadas dos conteúdos em cache na pasta static mesmo se você atualizar a página.
 
-### SPA Fallback
+### Aplicação de Página Única (SPA)
 
-Pages that have been excluded from generation, by using the `generate.exclude` property will fallback to being a single page application. These pages will therefore not exist in the CDN and will be rendered on client side in the browser once the user navigates to that page.
+As páginas que foram excluídas da geração, utilizando a propriedade `generation.exclude`, gerarão uma aplicação de página única. Essas páginas não existirão na CDN e serão renderizadas no lado do cliente no navegador uma vez que o usuário navegue até elas.
```
Comment (ChristopheCVB) [inline content/pt-br/docs/2.concepts/4.static-site-generation.md:31]: `generation.exclude` -> `generate.exclude`

File: content/pt-br/docs/2.concepts/5.nuxt-lifecycle.md [hunk 18]
```diff
@@ -1,108 +1,106 @@
...
-- The new fetch (top to bottom, siblings = parallel) (non-blocking)
-- beforeMount (Vue lifecycle method)
-- mounted (Vue lifecycle method)
+  - Middleware Global
+  - Middleware do Layout
+  - Middleware de Rotas
+- asyncData (bloqueio)
+- beforeCreate (Método do ciclo de vida do Vue)
+- created (Método do ciclo de vida do Vue)
+- Uma nova busca (de cima para baixo, em paralelo se forem vizinhos) (não bloqueio)
```
Comment (ChristopheCVB) [inline content/pt-br/docs/2.concepts/5.nuxt-lifecycle.md:81]: Uma nova busca -> O novo fetch

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]

File: content/pt-br/docs/2.concepts/5.nuxt-lifecycle.md (hunk 18 above)
Comment (felipesuri) [reply content/pt-br/docs/2.concepts/5.nuxt-lifecycle.md]: Solved, but you can explain more about `The new Fetch`, what is this? What does this do? So I can understand if there is another translation that fits better.

Comment (ChristopheCVB) [reply content/pt-br/docs/2.concepts/5.nuxt-lifecycle.md]: fetch is part of the lifecycle I believe, so no really something we want to translate

**Review by ChristopheCVB — APPROVED** [review]
👍

**Comment by smarroufin** [comment]
Thanks for your patience and your contribution @felipesuri 💚

**Pull request merged**